From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: zhang sanshan <pete.zhang@nxp.com>
Date: Wed, 24 Jun 2020 13:56:14 +0800
Subject: ANDROID: GKI: support CONFIG_INPUT_TOUCHSCREEN

enable build in driver drivers/input/touchscreen in gki configure
touch screen should been enable in gki configure which is need by vendor.

Bug: 159736147
Signed-off-by: zhang sanshan <pete.zhang@nxp.com>
Change-Id: I3b1d348791aa9e762374a4965893b3a4a490394b
---
 arch/arm64/configs/gki_defconfig | 1 +
 arch/x86/configs/gki_defconfig   | 1 +
 2 files changed, 2 insertions(+)

diff --git a/arch/arm64/configs/gki_defconfig b/arch/arm64/configs/gki_defconfig
--- a/arch/arm64/configs/gki_defconfig
+++ b/arch/arm64/configs/gki_defconfig
@@ -281,6 +281,7 @@ CONFIG_INPUT_EVDEV=y
 CONFIG_KEYBOARD_GPIO=y
 # CONFIG_INPUT_MOUSE is not set
 CONFIG_INPUT_JOYSTICK=y
+CONFIG_INPUT_TOUCHSCREEN=y
 CONFIG_INPUT_MISC=y
 CONFIG_INPUT_UINPUT=y
 # CONFIG_VT is not set
diff --git a/arch/x86/configs/gki_defconfig b/arch/x86/configs/gki_defconfig
--- a/arch/x86/configs/gki_defconfig
+++ b/arch/x86/configs/gki_defconfig
@@ -254,6 +254,7 @@ CONFIG_INPUT_EVDEV=y
 CONFIG_KEYBOARD_GPIO=y
 # CONFIG_INPUT_MOUSE is not set
 CONFIG_INPUT_JOYSTICK=y
+CONFIG_INPUT_TOUCHSCREEN=y
 CONFIG_INPUT_MISC=y
 CONFIG_INPUT_UINPUT=y
 # CONFIG_VT is not set
